backend/BobaList/query.py

```diff
@@ -1 +1,32 @@
 ## reserve for bobalist
+import graphene
+import json
+
+class BobaOptionMilkType(graphene.ObjectType):
+  name = graphene.String()
+
+class BobaOptionToppingsType(graphene.ObjectType):
+  name = graphene.String()
+  price = graphene.Float()
+
+class BobaOptionType(graphene.ObjectType):
+  milk = graphene.Field(lambda: BobaOptionMilkType)
+  toppings = graphene.List(lambda: BobaOptionToppingsType)
+
+class BobaType(graphene.ObjectType):
+  id = graphene.ID()
+  name = graphene.String()
+  description = graphene.String()
+  price = graphene.Float()
+  imageLink = graphene.String()
+  options = graphene.Field(lambda: BobaOptionType)
+
+class BobaQuery(graphene.ObjectType):
+  bobaList = graphene.List(BobaType)
+
+  def resolve_bobaList(self, info):
+    with open('./database/bobalist.json') as f:
```

Comment:
Does this work? I think the path is targeting wrong file

Reply:
Since it's './' which mean database is inside 'backend/BobaList'?
Are there more databases for ToppingList and MilkList?
We should only have one json file at root

Reply:
yes , the data base is the folder and have 3 .json files one for each, so u want to move everything under one file right( although i mean this will go away eventually after creating mongoDB )

Reply:
Stop referring that it will go away after MongoDB.
You should have single source of truth, one file.
Are you planning to delete 3 files from each folder? Why can't they be under one file to prevent duplicated files.